### PR TITLE
fix: liveness setup

### LINKED ISSUE #4: liveness probe does not set up properly

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Closes #4

### CODE REVIEW

File: src/common/interfaces.ts [hunk 1]
```diff
@@ -9,3 +9,7 @@ export interface OpenApiConfig {
   jsonPath: string;
   uiPath: string;
 }
+
+export interface IServerConfig {
+  port: string;
+}
```
Comment (CptSchnitz) [inline src/common/interfaces.ts]: just remove this interface :|

File: src/index.ts [hunk 2]
```diff
@@ -2,44 +2,60 @@
...
+    const livenessCheck = container.resolve<() => Promise<void>>(LIVENESS_PROBE_FACTORY);
+    container.resolve<void>(JOB_CLEANER_FACTORY);
 
-void getApp()
-  .then((app) => {
-    const logger = container.resolve<Logger>(SERVICES.LOGGER);
-    const stubHealthcheck = async (): Promise<void> => Promise.resolve();
+    const config = depContainer.resolve<IConfig>(SERVICES.CONFIG);
+    const serverConfig = config.get<IServerConfig>('server');
+    const port: number = parseInt(serverConfig.port) || DEFAULT_SERVER_PORT;
```
Comment (CptSchnitz) [inline src/index.ts]: change it so the value in the config is a number instead of parsing.

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

File: src/common/interfaces.ts (hunk 1 above)
Comment (syncush) [reply src/common/interfaces.ts]: done

File: src/index.ts (hunk 2 above)
Comment (syncush) [reply src/index.ts]: done

**Pull request merged**